modules/setup-bias-monitoring.adoc

```diff
@@ -0,0 +1,83 @@
+:_module-type: PROCEDURE
+
+[id="setup-bias-monitoring_{context}"]
+= Before you begin
+
+== Downloading tutorial files
+. Go to https://github.com/trustyai-explainability/odh-trustyai-demos/tree/main.
+. Click the *Code* button and then click *Download ZIP* to download the repository.
+. Extract the downloaded repository files.
+
+== Enabling user-workload-monitoring
+To get enable Open Data Hub’s monitoring stack, you must enable monitoring for user-defined projects.
+
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI:
+... In the upper-right corner of the OpenShift web console, click your user name and select *Copy login command*.
+... After you have logged in, click *Display token*.
+... Copy the *Log in with this token* command and paste it in the OpenShift command-line interface (CLI).
++
+[source]
+----
+oc login --token=<token> --server=<openshift_cluster_url>
+----
+
```

Comment:
== Enabling monitoring for user-defined projects
On the OpenShift cluster where {prodname-short} is installed, you must enable workload monitoring.
1.  Make sure your working directory is the directory that contains the downloaded tutorial files, for example:
cd odh-trustyai-demos-main
2. Run the following command to enable workload monitoring: